tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Thanks for fixing it related `+build` tag being deprecated since Go 1.17. Since the Go version set in go.mod is 1.18, it seems safe to delete the old comment.